x-pack/plugins/observability_ai_assistant/public/components/insight/insight_base.tsx

```diff
@@ -66,10 +69,18 @@ export function InsightBase({
               <AssistantAvatar size="xs" />
             </EuiFlexItem>
             <EuiFlexItem>
-              <EuiText css={{ marginTop: 2, marginBottom: 1 }}>
-                <h5>{title}</h5>
-              </EuiText>
+              <EuiFlexGroup gutterSize="s" alignItems="center">
+                <EuiText css={{ marginTop: 2, marginBottom: 1 }}>
+                  <h5>{title}</h5>
+                </EuiText>
 
+                <EuiIconTip
+                  content={
+                    'Every contextual insight can be changed with a custom prompt defined by the user. You can always reset it to the default.'
```

Comment:
This should be translated. 

Tip: if you in the IDE change `content` to `label` and save, the ESLint rule will auto add translation code. You can then change the prop back from `label` to `content`

Reply:
fixed in https://github.com/elastic/kibana/pull/176086/commits/3ef74e8b4784383775bfc0b38631a2bc18db8e6d